### PR TITLE
add gql query logging in orders service

### PR DESCRIPTION
**Pull request by ww-bennadler**
This was done only to orders service as an example, will add to the others after approved

### CODE REVIEW

File: orders-service/server.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
-const { ApolloServer, gql } = require('apollo-server');
+const { ApolloServer, gql } = require('apollo-server-express');
 const { buildSubgraphSchema } = require('@apollo/subgraph');
 const fs = require('fs');
+const express = require('express');
+const bodyParser = require('body-parser');
+
+
+// TODO - figure out how to use the logger
+// const logger = require('@ww/gql-base-service').logger.withContext(__filename);
```
Comment (adijesori) [inline orders-service/server.js]: redundant

File: gateway/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+GQL_TRACING=true
```
Comment (adijesori) [inline gateway/.env]: not sure we need it for the GW, let's put here false by default

File: gateway/index.js [hunk 3]
```diff
@@ -21,11 +26,33 @@ const serviceList = [
...
     supergraphSdl,
   });
+
   const server = new ApolloServer({
     schema,
   });
 
-  server.listen(4000).then(({ url }) => {
-    console.log(`🚀 Server ready at ${url}`);
+  if (process.env.GQL_TRACING) {
```
Comment (adijesori) [inline gateway/index.js:34]: Shouldn't it be like this?
```suggestion
  if (process.env.GQL_TRACING === 'true') {
```

**Review by adijesori — APPROVED** [review]
few small comments

**Comment by adijesori** [comment]
> This was done only to orders service as an example, will add to the others after approved

It seems you did it for all the services. No? (Same question for the PR title)
Also, can you share a picture of how it looks like?

File: gateway/index.js (hunk 3 above)
Comment (ww-bennadler) [reply gateway/index.js:34]: @adijesori seems to be working without, like in your real code

Comment (adijesori) [reply gateway/index.js:34]: yes, but now that you put `false` in GW, it will still print it. No?

**Pull request merged**